tags/README.md

```diff
@@ -1,86 +1,216 @@
-| TAG Contributor Strategy | [Stephen Augustus](https://github.com/justaugustus)      |
-| TAG Security             | [Brandon Lum](https://github.com/lumjjb)                 |
-| TAG Security             | [Aradhna Chetal](https://github.com/achetal01)           |
+<!-- Primary Authors: Alexis Richardson, Quinton Hoole
+
+with contributions by TOC Contributors, November 2018 - January 2019 
+
+v1.0
+
+Starting an update and revisit of the TAG folder, files and document content - Dec 2023 >
+-->
+## Introduction
+
+The purpose of this document is to provide insight into the objectives and life cycle of Technical Advisory Groups (TAGs) within the CNCF. Various TAGs exist, each within in the CNCF. More information about each TAG can found on the [TAG List](cncf-tags.md) and by following the links to the TAG's respective repositories.
+
+
+## Overall Purpose
+
+Scale contributions by the CNCF technical and user community, while retaining integrity and increasing quality in support of our [mission](https://github.com/cncf/foundation/blob/main/charter.md#1-mission-of-the-cloud-native-computing-foundation).
+
+## Specific Objectives
+
+* Strengthen the project ecosystem to meet the needs of end users and project contributors.
+* Identify gaps in the CNCF project portfolio.  Find and attract projects to fill these gaps.
+* Educate and inform users with unbiased, effective, and practically useful information.
+* Focus attention & resources on helping foster project maturity, systematically across CNCF projects.
+* Clarify relationship between projects, CNCF project staff, and community volunteers.
+* Engage more communities and create an on-ramp to effective TOC contribution & recognition.
+* Reduce some project workload on TOC while retaining executive control & tonal integrity with this elected body.
+* Avoid creating a platform for politics between vendors.
+* Provide a ladder for community members to get involved with the technical oversight of CNCF projects. As part of this, TAGs are expected to actively nurture diverse participation.
+
+## Introduction
+
+A CNCF TAG will oversee and coordinate the interests pertaining to a logical area of needs of end users and/or projects.  Examples of such areas include security, testing, observability, storage, networking, etc.  The area overseen by a TAG is typically met by a set of CNCF projects, and may also represent a cross-cutting feature group shared by several projects (like security and observability).  TAG’s are:
+
+* long lived groups that report to the Technical Oversight Committee
+* led primarily by recognised experts in the relevant field(s), supported by other contributors
+
+CNCF TAGs are modelled on Kubernetes SIGs.  Differences are intended to be minimal to avoid confusion - unavoidable differences are described [here](https://docs.google.com/document/d/1oSGhx5Hw7Hs_qawYB46BvRSPh0ZvFoxvHx-NWaf5Nsc/edit?usp=sharing).
+
```

Comment:
recommend moving this up under overall purpose.

Reply:
Agree, moved it.